lib/tasks/cleanup.rake

```diff
@@ -18,4 +18,17 @@ namespace :cleanup do
       Cart.destroy_all_instrument_only_carts(5.days.ago)
     end
   end
+
+  namespace :price_groups do
+    desc "remove cancer center"
+    task :cancer_center, [:cancer_center_name] => :environment do |_t, args|
+      cc_name = args[:cancer_center_name] || "Cancer Center Rate"
+      pg = PriceGroup.find_by(name: cc_name)
+      abort("No cancer center price group found with name #{cc_name}") unless pg
+
+      PriceGroup.skip_callback(:destroy, :before, :throw_abort)
```

Comment:
What about instead of skipping the callback, we set `pg.global = false`. Then the callback should get skipped as well and I would expect the destroy to go through. I think both feel a little hacky, but we are kind of hacking the system here.

Reply:
The `global?` method checks for an associated facility.  So the other option would be to set `pg.facility = Facility.first` or something like that.  Do we actually have a cancer center price group on UMass prod?  Another possible option would be adding a boolean to settings like `cancer_center_enabled` and adding some logic to the `global?` method:
 
```
before_destroy { throw :abort unless can_delete? }` 

def can_delete?
  # use !.any? because it uses SQL count(), unlike none?
  (is_not_global? || deletable_cancer_denter?) && !order_details.any?
end

def deletable_cancer_center?
  !Settings.cancer_center_enabled && name.include?("Cancer"))
end
```

If we do that, admins can delete the cancer center price group through the UI.

Reply:
There definitely is a cancer center group on UMass production. That's why we're trying to get rid of it: to prevent confusion when they go live for real.

If setting the `pg.facility = Facility.first` works and allows us to delete it without being to disruptive, I'd vote for that. This is a very special case so I'd rather not change internal functionality if we can avoid it. 

I'd prefer what you currently have with the skipping the callback over adding a new setting and allowing this to happen on the UI. I'll leave it up to you whether you want to go with the callback skipping or the facility setting.